src/modules/keyboardmanager/common/KeyboardManagerConstants.h

```diff
@@ -7,7 +7,7 @@
 namespace KeyboardManagerConstants
 {
     // Event name for signaling config changes

```

Comment:
`Event name for signaling config changes` -> `Event name for signaling settings changes`